candi/openapi/doc-ru-node_group.yaml

```
@@ -747,9 +747,9 @@ spec:
                       description: |
                         Управление менеджером топологии.
 
-                        Менеджер топологии позволяет планировщику Kubernetes использовать информацию о топологии ресурсов при планировании подов.
+                        Менеджер топологии помогает планировщику Kubernetes учитывать данные о топологии ресурсов при планировании подов.
```

```suggestion
                        Менеджер топологии помогает планировщику Kubernetes учитывать данные о топологии ресурсов при планировании подов.
```
@MariaMakeeva слово помогает тут не совсем подходит. т.е. без менеджера топологий планировщик вообще не учитывает топологию при планировании.